build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
     id("org.jlleitschuh.gradle.ktlint") version "9.2.1"
     id("org.sonarqube") version "2.8"
 
-    id("org.springframework.boot") version "2.2.4.RELEASE"
+    id("org.springframework.boot") version "2.3.1.RELEASE"
```

Comment:
Is there starter upraded here aswell? if not we should test this well before we release it.

Reply:
Base is on 2.3.0 afaik

Reply:
required due to dynamicpropertysource, and I agree. should be tested well